src/neo/SmartContract/ApplicationEngine.Contract.cs

```diff
@@ -18,6 +18,7 @@ partial class ApplicationEngine
         public static readonly InteropDescriptor System_Contract_Create = Register("System.Contract.Create", nameof(CreateContract), 0, CallFlags.AllowModifyStates, false);
         public static readonly InteropDescriptor System_Contract_Update = Register("System.Contract.Update", nameof(UpdateContract), 0, CallFlags.AllowModifyStates, false);
         public static readonly InteropDescriptor System_Contract_Destroy = Register("System.Contract.Destroy", nameof(DestroyContract), 0_01000000, CallFlags.AllowModifyStates, false);
+        public static readonly InteropDescriptor System_Contract_MethodExists = Register("System.Contract.MethodExists", nameof(MethodExists), 0_01000000, CallFlags.AllowCall, false);
```

Comment:
This SYSCALL is no longer needed. NEP-17 tokens call the `onPayment` method on the deployed contract directly. If the method doesn't exist, the transfer fails.

Reply:
We can call directly but I think that this syscall it can be useful for dapps.

Reply:
Dapps can read the ABI directly.

Reply:
Done